sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2858,6 +2860,20 @@ object Radar {
             })
     }
 
+    internal fun logOpenedAppConversion() {
+        // if opened_app has been logged in the last 1000 milliseconds, don't log it again
+        val timestamp = System.currentTimeMillis()
```

Comment:
`sessionId` already is the last app open timestamp, just (a) in seconds and (b) only updated if more than 5 minutes have elapsed (vs. 1 second)

Do we think we need to save a separate timestamp, or could we just look at `sessionId` instead?

Reply:
this is dedupe logic for if two app opens are sent in quick succession. It is not used in the same fashion that the session id timestamp is

Reply:
discussed [here](https://radarlabs.slack.com/archives/C02MF6PAWBH/p1679087562952389?thread_ts=1679086996.488419&cid=C02MF6PAWBH), tldr we want to make sure that we're capturing all app opens, so a 5 minute window is too long